websphere-liberty/content.md

```diff
@@ -2,14 +2,25 @@
 
 The images in this repository contain IBM WebSphere Application Server Liberty for Developers and the IBM Java Runtime Environment. See the license section below for restrictions relating to the use of this image. For more information about WebSphere Application Server Liberty, see the [WASdev](https://developer.ibm.com/wasdev/docs/category/getting-started/) site.
 
-# Images
+## Image User
 
-There are multiple images available in this repository. The image with the tag `beta` contains the contents of the install archive for the latest monthly beta. The other images are all based on the latest generally available fix pack.
+This image runs by default with `USER 1001` (non-root), as part of `group 0`.   All of the folders accessed by WebSphere Liberty been given the appropriate permission, but if your extending Dockerfile needs permission to another location you can simply temporarily switch into root and provide the needed permissions, example:
```

Comment:
```suggestion
This image runs by default with `USER 1001` (non-root), as part of `group 0`. All of the folders accessed by WebSphere Liberty been given the appropriate permission, but if your extending Dockerfile needs permission to another location you can simply temporarily switch into root and provide the needed permissions, example:
```

(There's a few extra "` `"s here that `markdownfmt` is still sad about.)

Reply:
:cry: it won't let me apply the suggestion. :confused:

Reply:
applied =)